v1.1/known-limitations.md

```diff
@@ -12,6 +12,8 @@ This page describes newly identified limitations in the CockroachDB v1.1 release
 
 ### Enterprise restores after creating, dropping, or truncating tables
 
+{{site.data.alerts.callout_info}}Resolved as of v1.1.1. It is no longer possible to create an incremental backup if a table has been been created, dropped, or truncated after the full backup. See <a href="https://github.com/cockroachdb/cockroach/pull/19286">#19286</a>.{{site.data.alerts.end}}
```

Comment:
I would say "changed" instead of "resolved". You still can't make a backup after dropping a table. The only change is that now it will tell you the backup has failed instead of leaving you with a broken backup and a false sense of security.

Reply:
Rephrased.